src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+        Select se = new Select(OrderTypeelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 0));
+
+    }
+    
+    private WebElement selectOrderType3;
+    /**
+     * Method to select Order Type.
+     * @throws IOException 
+     */
+    public void selectOrderType3() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        WebElement OrderTypeelement = driver.findElement(By.xpath("//select[@name='billingTypeId']"));
+        Select se = new Select(OrderTypeelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 1));
+
+    }
+    
+    private WebElement selectOrderType5;
+    /**
+     * Method to select Order Type.
+     * @throws IOException 
+     */
+    public void selectOrderType5() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        WebElement OrderTypeelement = driver.findElement(By.xpath("//select[@name='billingTypeId']"));
+        Select se = new Select(OrderTypeelement);
+        se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 1));
+
+    }
+    
+    private WebElement selectOrderType4;
+
+	/**
+	 * Method to select Order Type.
+	 * 
+	 * @throws IOException
+	 */
+	public void selectOrderType4() throws IOException {
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		JavaScriptExec.sleep();
+		WebElement OrderTypeelement = driver.findElement(By.xpath("//select[@name='billingTypeId']"));
+		Select se = new Select(OrderTypeelement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 5, 0));
+
+	}
+    
+    
+    
+    @FindBy(how=How.XPATH, using="//*[@id='ui-id-8']")
+    private WebElement clickProductsSubTab;
+    /**
+     * Method to Click Products tab.
+     * @throws IOException 
+     */
+    public void clickProductsSubTab() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+        log.info("Click Products Tab");
+        Assert.assertTrue(clickProductsSubTab.isDisplayed());
+        clickProductsSubTab.click();
+        JavaScriptExec.sleep();
+    }
+    
+    public void selectCategory() {
+		WebElement categoryElement = driver.findElement(By.xpath("//select[@name='typeId']"));
+		Select se1 = new Select(categoryElement);
+		se1.selectByVisibleText("Dependent Category");
+	}
+    
+    private WebElement selectProduct;
+    /**
+     * Method to select customer.
+     * @throws IOException 
+     */
+    public void selectProduct() throws IOException{
+    	CreateOrderPage sp=new CreateOrderPage(driver);
+    	JavaScriptExec.sleep();
+    	String ProductName = BasePage.getCellData(xlsxName, sheetName, 6, 0);
+		WebElement selectProduct = driver
+				.findElement(By.xpath("//a[@class='cell double']//*[text()='" + ProductName + "']"));
+		navigateBottom();
+		actions.moveToElement(selectProduct).click().perform();
+		JavaScriptExec.sleep();
+    }
```

Comment:
@mayankchhajer Again there are 3 sleeps in a method not good, Please organize them properly.